lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -793,7 +793,7 @@ class ClassMetadataInfo implements ClassMetadata
     /**
      * The ReflectionProperty instances of the mapped class.
      *
-     * @var array<string, ReflectionProperty|null>
+     * @var array<string, ReflectionProperty>
```

Comment:
Every time in the code, `$reflFields` is used this way
```
$reflFields['foo']->setValue()
```
so it would crash if null was in the array.

And looking at how the array is populated, we only set ReflectionProperty.